rep-0144.rst

```diff
@@ -102,7 +102,7 @@ Special Suffixes:
   ``pr2_robot``
 * a package containing the URDF and meshes of a robot should be named 
   ``<name_of_the_robot>_description``, e.g ``pr2_description``
-* if a package is meant for test only, prepend ``test``
+* if a package is meant for test only, append ``tests``
```

Comment:
Why use plural here? A lot of existing packages already follow the singular form, e.g. https://github.com/ros/ros_comm/tree/indigo-devel/test